message-channel/src/main/java/com/iluwatar/message/channel/App.java

```diff
@@ -0,0 +1,51 @@
+package com.iluwatar.message.channel;
+
+import org.apache.camel.CamelContext;
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.impl.DefaultCamelContext;
+
+/**
+ * 
+ * When two applications communicate with each other using a messaging system
+ * they first need to establish a communication channel that will carry the
+ * data. Message Channels are logical addresses in the messaging system.
+ * <p>
+ * The sending application doesn't necessarily know what particular application
```

Comment:
Yes. But if producers is lowercase, consumers should also be. Not a problem in the comment, but wouldn't be right on the code.